File: app/src/main/java/com/example/apprifa/Retrofit/RetrofitInit.java
package com.example.apprifa.Retrofit;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitInit {

    private final Retrofit retrofit;


    public RetrofitInit() {
        retrofit = new Retrofit.Builder()
                .baseUrl("http://ws.matheuscastiglioni.com.br/ws/")
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }

    public PostmonService getcep(){

        return retrofit.create(PostmonService.class);
    }
}

File: app/src/main/java/br/com/medeve/Retrofit/RetrofitInit.kt
package br.com.medeve.Retrofit

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

private val retrofit: Retrofit? = null
val URL: String = "https://viacep.com.br/ws/"

fun getcepService(retrofit: Retrofit) = retrofit.create(ICepService::class.java)!!


fun retrofitBuilder(): Retrofit {
    return Retrofit.Builder()
        .baseUrl(URL)
        .addConverterFactory(GsonConverterFactory.create())
        .build()

}